styles.css

```diff
@@ -0,0 +1,98 @@
+* {
+    box-sizing: border-box;
+}
+
+html {
+    scroll-behavior: smooth;
+}
+@font-face {
+    font-family: "PPTalisman Compact Bold";
+    src: url("font/PPTalisman-Compact-Extrabold.otf") format("opentype");
+}
+@font-face {
+    font-family: "PPTalisman Regular";
+    src: url("font/PPTalisman-Compact-Regular.otf") format("opentype");
+}
+@font-face {
+    font-family: "PPTalisman Condensed";
+    src:    url("font/PPTalisman-Condensed-Regular.otf") format("opentype");
+}
+
+body {
+    margin : 0;
+    font-family: "PPTalisman Condensed", sans-serif;
+    background-color: #e4e0e0;
+}
+
+nav {
+    display: flex;
+    justify-content: space-evenly;
+    background-color: chocolate;
+    padding: 0.5rem 0;
+}
+
+h1 {
+    font-family: "PPTalisman Compact Bold", sans-serif;
+
+}
+.highRatings h1, .library h1 {
+    text-align: center;
+    text-decoration: underline;
+    font-family: "PPTalisman Regular", sans-serif;
+}
+.hero div {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding-bottom: 2rem;
+    background-color: cornflowerblue;
+}
+
+
+
+.highRatings div, .library div{
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    align-content: center;
+    padding-top: 1rem;
+    padding-bottom: 0.5rem;
+    text-align: center;
+}
+
+img {
+    width: 176px;
+    height: 270px;
+}
+p {
+    margin: 0;
+}
+
+footer {
+    text-align: center;
+    font-size: 2rem;
+    padding-bottom: 1rem;
+}
+
+footer a{
+    text-decoration: none;
+    color: #000000;
+}
+footer a:hover{
+    color: orange;
+    text-decoration: underline;
+}
+
+@media screen and (min-width: 640px) {
+    .highRatings div, .library div {
+        flex-direction: row;
+        flex-wrap: wrap;
+        justify-content: center;
+        gap: 1rem;
+    }
+
+    .highRatings div div,.library div div {
```

Comment:
Deciding when to add classes to target divs is a bit of an art. While I did say it's preferable to target using nesting like this, sometimes that could be limiting if you want to make changes to the structure of your HTML. No need to change this, just something to bear in mind.